api/bibliotheca.py

```diff
@@ -1144,6 +1145,10 @@ class DateResponseParser(BibliothecaParser):
 
     def process_all(self, string):
         parser = etree.XMLParser()
+        # If the data is an HTTP response, it is a bytestring and
+        # must be converted before it is parsed.
+        if isinstance(string, bytes):
+            string = string.decode("utf-8")
```

Comment:
This makes me a bit queasy, but again its an upstream issue, so we can merge as is, then deal with it later. I think we should be figuring out where this is being called with a http response and instead using `response.text` instead of just doing a blind decode here.

https://docs.python-requests.org/en/master/user/quickstart/#response-content

Reply:
I agree that these will need to be addressed eventually. You and I discussed this (in a discussion on the related `circulation-core` PR, I think) and we decided that, in order to make merging from upstream easier, we would not try to address these piecemeal in the short tem unless we ran into a specific problem.

Reply:
Agreed. Was just making notes as I went through the PR. Don't think any of the encoding issues need to be dealt with here.